css/context-menu.css

```diff
@@ -0,0 +1,50 @@
+context-menu {
+    /* Variables */
+    --ctx-item-font-color: black;
```

Comment:
this should use css vars for colors coming from the [main palette](https://github.com/dkrasner/worksheet/blob/main/css/core.css#L1)

this will make it easier to change these uniformly, if the colors you want are not found as palette vars just add